### ISSUE TITLE
2FA page doesn't show correctly

### ISSUE THREAD

**Issue by DenisBY** [MagiusCHE/prospect-mail#9]
We have 2FA enabled to log in to the email. This is how the 2FA page looks like in 'original' 0.4.0:
![image](https://user-images.githubusercontent.com/1147703/152944541-e37bf761-9583-41b9-afcb-5e0dd08300fc.png)
And this is how it looks like in your 0.5.0-beta:
![image](https://user-images.githubusercontent.com/1147703/152944616-5304a7f9-c984-497d-a84d-2d54029cc27d.png)

Here is some output from dev console in 0.5.0-beta:
![image](https://user-images.githubusercontent.com/1147703/152944698-9bbb86d9-0cdf-44bd-830e-1e9e1c58318a.png)

Just in case this is the same output for 0.4.0:
![image](https://user-images.githubusercontent.com/1147703/152944803-013ae66c-f7df-4a2b-b58c-02791977f5d6.png)

Let me know if you need more debug info

**Comment by MagiusCHE** [comment]
@DenisBY i need a deeper log about this. Can you attach your `har` and `log`?
To produce `*.log` you have to point to Console and right-click and Save as
![immagine](https://user-images.githubusercontent.com/46496052/153281946-fc32d9a6-2e5b-474a-bc70-7088ef100dd4.png)
To produce `*.har` you have to go in Network tab and click on `Export HAR`
![immagine](https://user-images.githubusercontent.com/46496052/153282030-67c04194-5fb1-43cc-a87d-0d80d4b16785.png)

I'll try to figure out something from them.

**Comment by DenisBY** [comment]
Clicking on "Save as..." and Arrow Down from your screenshots don't do anything. No new windows opened. Or does it save somewhere automatically?

**Comment by galvianistudioeco** [comment]
It should save files into default download directory.
Try and search them named *.har, *.log

**Comment by MagiusCHE** [comment]
It should save files into default download directory.
Try and search them named *.har, *.log

**Comment by DenisBY** [comment]
No, no any file like *.har. I guess on clicking "Save as..." there should be a new window opened where I can select where to save the logs. But no any window appears. Clicking "Save as..." or arrow down literally does nothing.

**Comment by MagiusCHE** [comment]
@DenisBY you're right!
So i have to figure out how to setup 2FA to a brand new account and try it.
I will try to asap.
Do you know if normal user can activate this feature without be in an organization?

**Comment by DenisBY** [comment]
Sorry, I don't know.

**Comment by MagiusCHE** [comment]
@DenisBY ok... i've done a step forward.
Due to this bug https://github.com/electron/electron/issues/31152 you cant "save as" required log.
I've just recompiled and published a new version here https://github.com/MagiusCHE/prospect-mail/releases with this Electron bug officially fixed.
Can you try to install it and grab the logs now?

**Comment by DenisBY** [comment]
here it is:
```
jquery-migrate-1.2.1.min.js:2 Uncaught ReferenceError: jQuery is not defined
    at jquery-migrate-1.2.1.min.js:2
(anonymous) @ jquery-migrate-1.2.1.min.js:2
jquery.poshytip.min.js:7 Uncaught ReferenceError: jQuery is not defined
    at jquery.poshytip.min.js:7
(anonymous) @ jquery.poshytip.min.js:7
bootstrap.min.js:6 Uncaught Error: Bootstrap requires jQuery
    at bootstrap.min.js:6
(anonymous) @ bootstrap.min.js:6
jquery.qtip.js:24 Uncaught ReferenceError: jQuery is not defined
    at jquery.qtip.js:24
    at jquery.qtip.js:28
    at jquery.qtip.js:3063
(anonymous) @ jquery.qtip.js:24
(anonymous) @ jquery.qtip.js:28
(anonymous) @ jquery.qtip.js:3063
angular.js:11655 TypeError: a.Spinner is not a constructor
    at Object.<anonymous> (angular-spinner.min.js:1)
    at Object.fn (angular.js:12895)
    at n.$digest (angular.js:14308)
    at n.$apply (angular.js:14571)
    at angular.js:1455
    at Object.e [as invoke] (angular.js:4203)
    at d (angular.js:1453)
    at uc (angular.js:1473)
    at Jd (angular.js:1367)
    at angular.js:26304
(anonymous) @ angular.js:11655
error @ js:2365
$digest @ angular.js:14326
$apply @ angular.js:14571
(anonymous) @ angular.js:1455
e @ angular.js:4203
d @ angular.js:1453
uc @ angular.js:1473
Jd @ angular.js:1367
(anonymous) @ angular.js:26304
a @ angular.js:2762
c @ angular.js:3032
angular.js:11655 Error server-side logging failed
(anonymous) @ angular.js:11655
error @ js:2389
$digest @ angular.js:14326
$apply @ angular.js:14571
(anonymous) @ angular.js:1455
e @ angular.js:4203
d @ angular.js:1453
uc @ angular.js:1473
Jd @ angular.js:1367
(anonymous) @ angular.js:26304
a @ angular.js:2762
c @ angular.js:3032
angular.js:11655 ReferenceError: $ is not defined
    at error (js:2376)
    at n.$digest (angular.js:14326)
    at n.$apply (angular.js:14571)
    at angular.js:1455
    at Object.e [as invoke] (angular.js:4203)
    at d (angular.js:1453)
    at uc (angular.js:1473)
    at Jd (angular.js:1367)
    at angular.js:26304
    at HTMLDocument.a (angular.js:2762)
VM250:188 Missing valid handler, try again in 5 seconds
VM250:188 Missing valid handler, try again in 5 seconds
VM250:188 Missing valid handler, try again in 5 seconds
VM250:188 Missing valid handler, try again in 5 seconds
```

**Comment by MagiusCHE** [comment]
Please attach HAR too.

**Comment by DenisBY** [comment]
Here it is.
https://file.io/oeAsvhb9bNkv
This is a "one-time" share. The file will be deleted after the first download.

**Comment by MagiusCHE** [comment]
@DenisBY so the problem is this: i cant reprocude your bug because you are using a custom 2FA and i cant try it (the default one works like a charm).
Your logs displays an error based of missing jquery and no other "prospect-mail" specific error.
I try to analyze it checking the injecting method but, i repeat, if i cant reproduce your case i cant fix it.

Sorry but it will be hard.

I just tried to point to https://login.vip.symantec.com/viplogin/jwt/2fa any my prospect-mail show this:
![immagine](https://user-images.githubusercontent.com/46496052/154572031-2c3eca1b-2d99-414e-87e8-b7dd9f1b6928.png)

I guess i cant use it but it shows something (not like your screen that is seems blank).

**Comment by DenisBY** [comment]
I see. Thank you for your efforts anyway. Maybe you can examine your changes compared with 0.4.0 as it's working without any issues?

**Comment by MagiusCHE** [comment]
But, just to be sure: have you tried to call the url directly from browser right? does it work?

**Comment by MagiusCHE** [comment]
But, just to be sure: have you tried to call the url directly from browser right? does it work?
i meant: directly form CHROME or CHROMIUM browser

**Comment by DenisBY** [comment]
And Firefox as well. No issues at all.

**Comment by MagiusCHE** [comment]
Well i've added a command line `--disablecontextmenu` to prevent the only relevant change from 0.4.0 to 0.5.0.
What package do you use? (.pacman? .deb? or?)
I will upload it so you can try run client passing `--disablecontextmenu` as command line arg.

**Comment by DenisBY** [comment]
pacman

**Comment by MagiusCHE** [comment]
published here https://github.com/MagiusCHE/prospect-mail/releases

**Comment by DenisBY** [comment]
Same problem with `--disablecontextmenu`

```
$ prospect-mail --disablecontextmenu
Loaded settings {
  outlookUrl: 'https://outlook.office.com/mail',
  deeplinkUrls: [
    'outlook.live.com/mail/deeplink',
    'outlook.office365.com/mail/deeplink',
    'outlook.office.com/mail/deeplink',
    'outlook.office.com/calendar/deeplink'
  ],
  outlookUrls: [ 'outlook.live.com', 'outlook.office365.com', 'outlook.office.com' ]
}
ContextMenu disabled via --disablecontextmenu cmdarg.
libva error: /usr/lib/dri/iHD_drv_video.so init failed
[365241:0224/092613.250267:ERROR:sandbox_linux.cc(376)] InitializeSandbox() called with multiple threads in process gpu-process.
Prepare 'main.css' to be injected.
Prepare 'unread-number-observer.js' to be injected.
```

**Comment by julian-alarcon** [comment]
Can you try with the latest beta version (0.5-beta1)?
`sudo snap install prospect-mail --channel=beta`

Also available here in other packages: https://github.com/julian-alarcon/prospect-mail/releases/tag/v0.5.0-beta1

**Comment by DenisBY** [comment]
I tried but it has the same issue.
